File: Programmers/Swift/피보나치 수.swift
let MAX = 1000001
let MOD = 1234567

func solution(_ n:Int) -> Int {
    var a = [Int](repeating: 0, count: MAX)
    a[1]=1
    for i in 2...n {
        a[i]=(a[i-1]+a[i-2])%MOD
    }
    return a[n]
}

File: Programmers/C++/피보나치 수.cpp
#include <string>
#include <vector>
#define FOR(i,n) for(int i=0;i<(n);++i)
#define FORS(i,s,n) for(int i=(s);i<(n);++i)
#define MAX 100001
#define MOD 1234567
using namespace std;

int a[MAX]={0,1,};

int solution(int n) {
    FORS(i,2,n+1) a[i]=(a[i-1]+a[i-2])%MOD;
    return a[n];
}
